airbyte-integrations/connectors/source-weather/source_weather/source.py

```diff
@@ -0,0 +1,228 @@
+        This means that data in a slice is usually closely related to a stream's cursor_field and stream_state.
+
+        An HTTP request is made for each returned slice. The same slice can be accessed in the path, request_params and request_header functions to help
+        craft that specific request.
+
+        For example, if https://example-api.com/v1/employees offers a date query params that returns data for that particular day, one way to implement
+        this would be to consult the stream state object for the last synced date, then return a slice containing each date from the last synced date
+        till now. The request_params function would then grab the date from the stream_slice and make it part of the request by injecting it into
+        the date query param.
+        """
+        raise NotImplementedError("Implement stream slices or delete this method!")
+
+
+# Source
+class SourceWeather(AbstractSource):
+
+    def check_connection(self, logger, config) -> Tuple[bool, any]:
+        """
+        TODO: Implement a connection check to validate that the user-provided config can be used to connect to the underlying API
+
+        See https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-stripe/source_stripe/source.py#L232
+        for an example.
+
+        :param config:  the user-input config object conforming to the connector's spec.yaml
+        :param logger:  logger object
+        :return Tuple[bool, any]: (True, None) if the input config can be used to connect to the API successfully, (False, error) otherwise.
+        """
+        if "lat" not in config or "lon" not in config and "apikey" not in config:
+            log("Input config must contain the properties 'api_key' and 'lat'"and 'lon')
+            sys.exit(1)
+        else:
+        # Validate input configuration
+            response = requests.get('https://api.openweathermap.org/data/2.5/weather', params=config)
+            if response.status_code == 200:
+                result = {"status": "SUCCEEDED"}
+            elif response.status_code == 403:
+                # HTTP code 403 means authorization failed so the API key is incorrect
+                result = {"status": "FAILED", "message": "API Key is incorrect."}
+                return False, None
+            else:
+                # Consider any other code a "generic" failure and tell the user to make sure their config is correct.
+                result = {"status": "FAILED", "message": "Input configuration is incorrect. Please verify the input stock ticker and API key."}
+                return False, None
+            # Format the result of the check operation according to the Airbyte Specification
+            output_message = {"type": "CONNECTION_STATUS", "connectionStatus": result}
+            print(json.dumps(output_message))
```

Comment:
Please carefully read API of `check_connection` method
you don't need to `print` any json 
you just need to `return True, None` if all OK
or `return False, error_message` in case of error